workflow/scripts/extract-personal-data.py

```diff
@@ -67,7 +65,10 @@ def save_personal_data(personal_data: dict, out_path: str):
     # if personal_data.get("country"):
     #     personal_data = format_country(personal_data)
 
-    save_personal_data(personal_data, out_path)
+    personal_data = {key: value.lower() for key, value in personal_data.items()}
+    personal_data = {key: value.strip() for key, value in personal_data.items()}
```

Comment:
Müsste man so zusammenfassen können:
personal_data = {key: value.lower().strip() for key, value in personal_data.items()}

Reply:
👍   changed this in 503d12f